docs/apache-airflow-providers-amazon/operators/transfer/s3_to_sql.rst

```diff
@@ -0,0 +1,75 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+============================
+Amazon S3 to SQL
+============================
+
+Use the ``S3ToSqlOperator`` transfer to copy data from an Amazon Simple Storage Service (S3) file into an existing
+SQL table.
+Only CSV Format is supported.
```

Comment:
Should the operator fail fast and prior to downloading from S3 if the file is not a CSV? Can this be implied by the `s3_key` value?

If you agree, then let's add a test for this situation too.